healthtools/config.py

```diff
@@ -22,16 +22,16 @@
 SMALL_BATCH_HF = 100  # No of records scraped from health-facilities sites in development mode
 SMALL_BATCH_NHIF = 1  # No of nhif accredited facilities scraped in development mode
 
-# Where we archive the data
-DATA_DIR = os.getcwd() + "/data/"
-if not os.path.exists(DATA_DIR):
-    os.mkdir(DATA_DIR)
-    os.mkdir(DATA_DIR + "archive")
-    os.mkdir(DATA_DIR + "test")
-    os.mkdir(DATA_DIR + "test/archive")
-
 if AWS["s3_bucket"]:
     DATA_DIR = "data/"
+else:
+    # Where we archive the data in case of no s3 Bucket
+    DATA_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__))) + "/data/"
+    if not os.path.exists(DATA_DIR):
+        os.mkdir(DATA_DIR)
+        os.mkdir(DATA_DIR + "archive")
+        os.mkdir(DATA_DIR + "test")
+        os.mkdir(DATA_DIR + "test/archive")
```

Comment:
@DavidLemayian how about we create a method that is called while the scraper is initializing that creates the key if non-existent so that only what someone needs is just a valid bucket and we create the objects